src/app/components/case/Documents/DocumentsTable/DocumentsTable.tsx

```diff
@@ -1,35 +1,34 @@
 import { useEffect, useMemo, useState } from "react"
 import { Table } from "@amsterdam/wonen-ui"
-import { useDocumentTypesByCase } from "app/state/rest"
 import getColumns from "./columns"
 
 type Props = {
   caseId: Components.Schemas.CaseEvent["id"]
-  data?: Components.Schemas.DocumentType[]
+  documents?: Components.Schemas.PaginatedDocumentTypeList["results"]
   loading: boolean
   getDocuments: () => Promise<unknown>
+  documentTypes?: Components.Schemas.DocumentType[]
 }
 
-const DocumentsTable: React.FC<Props> = ({ caseId, data, loading, getDocuments }) => {
+const DocumentsTable: React.FC<Props> = ({ caseId, documents, loading, getDocuments, documentTypes }) => {
   const [numLoadingRows, setNumLoadingRows] = useState(3)
-  const [documentTypes] = useDocumentTypesByCase(caseId)
 
   useEffect(() => {
-    if (data && data?.length > 0) {
+    if (documents && documents?.length > 0) {
       // Set number of loading rows to keep the table height when loading
-      setNumLoadingRows(data.length)
+      setNumLoadingRows(documents.length)
     }
-  }, [data])
+  }, [documents])
 
   const columns = getColumns(getDocuments, documentTypes)
 
-  // By sorting data for id, records are sorted for creation date.
-  const sortedData = useMemo(() => data?.sort((a: any, b: any) => b?.id - a?.id), [data])
+  // By sorting documents for id, records are sorted for creation date.
+  const sortedDocuments = useMemo(() => documents?.sort((a: any, b: any) => b?.id - a?.id), [documents])
```

Comment:
Al die vraagtekens (a?.id en b?.id) voelen een beetje vreemd en zijn misschien onnodig, maar ik weet te weinig over het product om hier echt iets van te vinden 
Is waarschijnlijk ook allemaal niet zo erg als het blijft dus kan je prima negeren. 🤷

Reply:
Klopt ook! 😉 Documents komt van open-zaak en is niet scherp gedefinieerd. Ik wil daar niet afhankelijk van zijn en daarom de vraagtekens. Als open-zaak nu iets anders terug stuurt, crasht de app niet.